pklid.go

```diff
@@ -0,0 +1,35 @@
+package nmea
+
+const (
+	// TypePKLID type for PKLID sentances
+	TypePKLID = "KLID"
+)
+
+
+// PKLID is a Kenwood Propritary sentance used for GPS data communications in FleetSync.
+// $PKLID,<0>,<1>,<2>,<3>,<4>*hh<CR><LF>
+// Format:  $PKLID,xx,xxx,xxxx,xx,xx,*xx<CR><LF>
+// Example: $PKLID,00,100,2000,15,00,*??
+type PKLID struct {
+	BaseSentence
+	SentanceVersion	string	// 00 to 15
+	Fleet		string	// 100 to 349
+	UnitID		string	// 1000 to 4999
+	Status		string	// 10 to 99
+	Extension	string	// 00 to 99
```

Comment:
Are we sure these should be strings instead of integers?

Reply:
I believe they should be strings, this would keep it consistent with how the pynema python library was handling them.   The notes there are based on what I've observed from captures , I was trying to keep them consistent across the different sentences, this is particularly noticeable on UnitID which on some sentences types starts with the letter U